src/Markdig.Tests/Specs/GridTableSpecs.md

```diff
@@ -70,8 +70,8 @@ A regular row can continue a previous regular row when column separator `|` are
 +---------+---------+---------+
 | Col1    | Col2    | Col3    |
 | Col1a   | Col2a   | Col3a   |
-| Col12             | Col3b   |
-| Col123                      |
+| Col1b             | Col3b   |
```

Comment:
Don't forget to regenerate the Specs.cs by saving the tt again (you can amend this commit)